runtime/drivers/duckdb/connectors_test.go

```diff
@@ -5,10 +5,14 @@ import (
 	"testing"
 
 	"github.com/rilldata/rill/runtime/connectors"
+	_ "github.com/rilldata/rill/runtime/connectors/gcs"
+	_ "github.com/rilldata/rill/runtime/connectors/s3"
 	"github.com/rilldata/rill/runtime/drivers"
 	"github.com/stretchr/testify/require"
 )
 
+const pathPrefix = "../../../web-local/test/data/"
```

Comment:
1. Maybe call this `testdataPath` instead?
2. Should we move test data to the runtime? We could put it in `runtime/pkg/testdata` (`testdata` is a special name in Go that's ignored by the build tool)

Reply:
Ya will add the test data and the generator code. Was planning to do this in the glob PR since it will need some more generated data.